x-pack/plugin/eql/src/test/resources/queryfolder_tests.txt

```diff
@@ -206,22 +206,37 @@ InternalEqlScriptUtils.concat([InternalQlScriptUtils.docValue(doc,params.v0),par
 cidrMatchFunctionOne
 process where cidrMatch(source_address, "10.0.0.0/8")
 ;
-"term":{"source_address":{"value":"10.0.0.0/8"
+{"bool":{"must":[{"term":{"event.category":{"value":"process"
+{"terms":{"source_address":["10.0.0.0/8"]
+;
+
+cidrMatchFunctionOneBool
+process where cidrMatch(source_address, "10.0.0.0/8") == true
+;
+{"bool":{"must":[{"term":{"event.category":{"value":"process"
+{"terms":{"source_address":["10.0.0.0/8"]
 ;
 
 cidrMatchFunctionTwo
```

Comment:
Please, add one more test where there are two `cidrMatch` function calls in the same query: `where cidrMatch(whatever) or cidrMatch(another_whatever)`.